tests/unit/core/utils/test_index.js

```diff
@@ -792,4 +797,81 @@ describe(__filename, () => {
       });
     });
   });
+
+  describe('getPreviewImage', () => {
+    it('returns the first full image from the previews array', () => {
+      const fullImage = 'https://addons.cdn.mozilla.net/full/12345.png';
+      const addon = createInternalAddon({
+        previews: [
+          {
+            ...fakePreview,
+            image_url: fullImage,
+          },
+          {
+            ...fakePreview,
+            image_url: 'https://addons.cdn.mozilla.net/image.not.used.here.png',
+          },
+        ],
+      });
+      const image = getPreviewImage(addon);
+      expect(image).toEqual(fullImage);
+    });
+
+    it('returns the full image from the 2nd item in the previews array', () => {
```

Comment:
I get it but this is weird to see `2nd` in the description and `version: 1` in the assertion.

Reply:
hmm ok..let me try something else